python/ray/tune/hyperband.py

```diff
@@ -372,7 +376,9 @@ def __repr__(self):
         status = ", ".join([
             "n={}".format(self._n),
             "r={}".format(self._r),
-            "progress={}".format(self.completion_percentage())
+            "completed={}%".format(int(100 * self.completion_percentage()))
             ])
-        trials = ", ".join([t.status for t in self._live_trials])
-        return "Bracket({})[{}]".format(status, trials)
+        counts = collections.defaultdict(int)
```

Comment:
there is `Counter` from collections that does the same thing, by the way

Reply:
Changed